contracts/contracts/multisig/Multisig.cairo

```diff
@@ -383,23 +358,21 @@ func submit_transaction{
 end
 
 @external
-func confirm_transaction{
-        syscall_ptr : felt*,
-        pedersen_ptr : HashBuiltin*,
-        range_check_ptr
-    }(tx_index : felt):
+func confirm_transaction{syscall_ptr : felt*, pedersen_ptr : HashBuiltin*, range_check_ptr}(
+    tx_index : felt
+):
     require_owner()
     require_tx_exists(tx_index=tx_index)
     require_not_executed(tx_index=tx_index)
+    require_valid_tx(tx_index=tx_index)
+    # TODO: clarify if needed
```

Comment:
I'd suggest renaming as `require_tx_valid`, switching order (both to be more consistent and readable), then add necessary inline comments to the `require_tx_valid` fn about what constitutes a valid tx.

```suggestion
    require_tx_exists(tx_index=tx_index)
    require_tx_valid(tx_index=tx_index)
    require_not_executed(tx_index=tx_index)
```